src/tools/tidy/src/deps.rs

```diff
@@ -67,6 +67,7 @@ pub(crate) const WORKSPACES: &[(&str, ExceptionList, Option<(&[&str], &[&str])>)
     //("src/tools/miri/test-cargo-miri", &[], None), // FIXME uncomment once all deps are vendored
     //("src/tools/miri/test_dependencies", &[], None), // FIXME uncomment once all deps are vendored
     ("src/tools/rust-analyzer", EXCEPTIONS_RUST_ANALYZER, None),
+    ("src/tools/rustc-perf", EXCEPTIONS_RUSTC_PERF, None),
```

Comment:
> davidtwco and I talked about this today and we think it makes sense to add the `EXCEPTIONS_RUSTC_PERF` list and we're ok with the current set of crates/licenses in it. It would be best if we can avoid growing the list further, just to make maintenance easier but this seems like the correct approach to us. 

> I'm not sure if the list contains everything needed currently though. Does it contain just the dependencies of `rustc-perf`, or also the dependencies of the compile-time benchmarks that we use for the PGO workflow?

@wesleywiser @Kobzol 

It currently only includes dependencies of rustc-perf, as those training data crates are not part of the rustc-perf workspace (also the reason we manually `cargo vendor` them). Should we license-check those training data crates as well?

(moved to a thread for following the discussion easier)

Reply:
I *think* that we should check the licenses of everything that we vendor, but I'm not 100% sure.

Reply:
> check the licenses of everything that we vendor

Including the web frontend code? 🤯 
https://github.com/rust-lang/rustc-perf/blob/master/site/frontend/package.json

Reply:
My assumption was that we should check licences of everything that we vendor, although I'm not sure if we currently do that. If that is needed, it would probably be the simplest if we just deleted the frontend code before packaging it in the source tarball.